tensorflow/core/util/gpu_launch_config.h

```diff
@@ -326,11 +326,11 @@ inline Gpu3DLaunchConfig GetGpu3DLaunchConfig(
   //CHECK_EQ(err, hipSuccess);
```

Comment:
HIP doesn't support `hipOccupancyMaxPotentialBlockSize` yet so code paths here can't be merged